packages/service-clients/azure-client/src/AzureAudience.ts

```diff
@@ -19,7 +19,7 @@ export function createAzureAudienceMember(audienceMember: IClient): AzureMember
 
 	return {
 		userId: user.id,
```

Comment:
While we're at it, maybe we should rename userId to id, to match the properties we receive from FRS?